### ISSUE TITLE
OneCycleLR argument `pct_start` used as a proportion not percentage

### ISSUE THREAD

**Issue by tmassingham-ont** [pytorch/pytorch#42447]
## 📚 Documentation

The documentation for the `OneCycleLR` learning rate schedule refers to the argument `pct_start` as being a percentage [0,100], whereas it actually used as a proportion [0,1] in the code.

> pct_start (float) – The percentage of the cycle (in number of steps) spent increasing the learning rate. Default: 0.3

compared to the implementation in `torch.optim.lr_scheduler.OneCycleLR`
```python
        self.step_size_up = float(pct_start * self.total_steps) - 1
```
